src/StoreKeeper/WooCommerce/B2C/Imports/ProductImport.php

```diff
@@ -1039,11 +1039,74 @@ function ($attribute) {
+                function ($attribute) {
+                    return array_key_exists('attribute_option_color_hex', $attribute);
+                }
+            );
+
+            $mergedAttributes = array_merge($nonAttributeOptions, $colorAttributes);
+
+            $blocksyTaxonomyMetaOptions = [];
+
+            foreach ($mergedAttributes as $attribute) {
                 if (key_exists('attribute_id', $attribute)) {
+                    $BlogModule = $this->storekeeper_api->getModule('BlogModule');
+                    $response = $BlogModule->listTranslatedAttributes(
+                        0,
+                        0,
+                        1,
+                        [
+                            [
+                                'name' => 'id',
+                                'dir' => 'desc',
+                            ],
+                        ],
+                        [
+                            [
+                                'name' => 'id__=',
+                                'val' => $attribute['attribute_id'],
+                            ],
+                        ]
+                    );
+
+                    $data = $response['data'][0];
+                    $attributeType = $data['type'];
                     $label = sanitize_title($attribute['label']);
-                    $attribute_id = $attribute['attribute_id'];
-                    update_post_meta($newProduct->get_id(), 'attribute_id_'.$attribute_id, $label);
+                    $attributeId = $attribute['attribute_id'];
+
+                    if ($attributeType === 'color') {
+                        $blocksyTaxonomyMetaOptions['color_type'] = 'simple';
+                        $blocksyTaxonomyMetaOptions['accent_color'] = [
+                            'default' => ['color' => $attribute['attribute_option_color_hex']],
+                            'secondary' => ['color' => 'CT_CSS_SKIP_RULE']
+                        ];
+                    } else {
+                        $blocksyTaxonomyMetaOptions['color_type'] = 'default';
+                    }
+
+                    $blocksyTaxonomyMetaOptions['tooltip_type'] = 'default';
+                    $blocksyTaxonomyMetaOptions['tooltip_mask'] = '{term_name}';
+                    $blocksyTaxonomyMetaOptions['tooltip_image'] = '';
+
+                    $serializedMetaOptions = serialize($blocksyTaxonomyMetaOptions);
+
+                    update_post_meta($newProduct->get_id(), 'blocksy_taxonomy_meta_options', $blocksyTaxonomyMetaOptions);
+
+                    update_post_meta($newProduct->get_id(), 'attribute_id_' . $attributeId, $label);
+
+                    if (key_exists('value_label', $attribute)) {
+                        $valueLabel = $attribute['value_label'];
+                        update_post_meta($newProduct->get_id(), 'value_label_' . $attributeId, $valueLabel);
+                        $term = term_exists($valueLabel, 'pa_' . sanitize_title($attribute['label']));
```

Comment:
that does not look correct. Read the term id using `src/StoreKeeper/WooCommerce/B2C/Models/AttributeOptionModel.php`